winlogbeat/etc/fields.yml

```diff
@@ -228,7 +228,7 @@ eventlog:
       description: The version number of the event's definition.
 
     - name: xml
-      type: string
```

Comment:
We could probably raise the limit just for this one here. I set it text as I expected that it would be nice to do "full text search" on the xml string? So this string is more used as additional info but not for query purpose?